recipe/meta.yaml

```diff
@@ -28,10 +24,10 @@ requirements:
     - {{ compiler('cuda') }}
   host:
     - python
+    - pip
+    - packaging
```

Comment:
`packaging` is missing from `pyproject.toml` upstream and it is a known issue ( https://github.com/NVIDIA/apex/issues/1679 )